Source/DafnyCore/Verifier/Translator.ClassMembers.cs

```diff
@@ -1334,7 +1370,58 @@ public partial class Translator {
       builder.Add(Assert(original.Tok, decrChk, new PODesc.TraitDecreases(original.WhatKind)));
     }
 
-    private void AddMethodOverrideSubsetChk(Method m, BoogieStmtListBuilder builder, ExpressionTranslator etran, List<Variable> localVariables,
+    // TODO: Worth sharing code between this and AddMethodOverrideModifiesSubsetChk?
+    private void AddMethodOverrideReadsSubsetChk(Method m, BoogieStmtListBuilder builder, ExpressionTranslator etran, List<Variable> localVariables,
```

Comment:
Could you clean up the code in the method `AddMethodOverrideReadsSubsetChk` a bit? It shows up as new in this PR but I see it was existing code.

Reply:
Yup I'll first seem if I can merge the two, and either way clean them up.